pokete.py

```diff
@@ -572,20 +674,34 @@ def has_item(self, item):
             return False
 
     def remove_item(self, item, amount=1):
+        """Removes a certain amount of an item from the inv"""
         assert amount > 0, "Amounts have to be positive"
         assert item in self.inv, f"Item {name} is not in the inventory"
-        assert self.inv[item]-amount >= 0, f"There are not enought {name}s in the inventory"
+        assert self.inv[item] - amount >= 0, f"There are not enought {name}s in the inventory"
```

Comment:
- Unresolved Reference 'name'
- Also: Typo
```suggestion
        assert self.inv[item] - amount >= 0, f"There are not enough {item}s in the inventory"
```

Reply:
Found the same issue in line 679.

Reply:
Yes. Was currently trying to get a new suggestion going, but gh didn't want to...

Reply:
Anyway, I fixed it.